backend/src/govsg/services/govsg-callback.service.ts

```diff
@@ -226,6 +283,42 @@ const parseTemplateMessageWebhook = async (
       void govsgMessage?.update(fieldOpts, whereOpts)
       void govsgMessageTransactional?.update(fieldOpts, whereOpts)
       void govsgOp?.update(fieldOpts, whereOpts)
+      // TODO: Add logging statement here for traceability
+      if (!govsgMessage && !govsgOp) {
+        return
+      }
+      const message = (govsgMessage ?? govsgOp) as GovsgMessage
+      // TODO: Add logging statement here for traceability
+      const experimentalData = await experimentService.getUserExperimentalData(
+        message.campaign.userId
+      )
+      // TODO: Add logging statement here for traceability
+      const canAccessGovsgV = `${ChannelType.Govsg}V` in experimentalData
+      if (!canAccessGovsgV) {
+        return
```

Comment:
Similar to above

Reply:
This one is fine because it just means the user ain't got access to our new feature. It's a bit too bad for them.